README.md

```diff
@@ -1,8 +1,25 @@
 # codeartifact-gradle-plugin
 
-A set of plugins for authenticating with and using [AWS CodeArtifact](https://aws.amazon.com/codeartifact/) repository as a source for project plugins 
+[![Gradle Plugin Portal](https://img.shields.io/maven-metadata/v/https/plugins.gradle.org/m2/co/bound/plugins/maven-metadata.xml.svg?colorB=007ec6&label=Plugin%20Portal)](https://plugins.gradle.org/plugin/co.bound.codeartifact)
+
+A set of plugins for authenticating with and using [AWS CodeArtifact](https://aws.amazon.com/codeartifact/) repository as a source for project plugins
 and dependencies as well as target for publishing artifacts to.
 
+## Why another CodeArtifact plugin?
+
+The existing solutions for Gradle with CodeArtifact cater for the most common use cases - consuming and publishing
+project dependencies from/to CodeArtifact repository.
+
+Another use case exists though - organizations that work with multiple projects scattered in multiple repositories may want
```

Comment:
"work with multiple projects spread across many repositories"? "Scattered" sounds messy and chaotic, which is probably not the impression we want to give 😄